.buildkite/pipeline.sh

```diff
@@ -1,10 +1,34 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -euo pipefail
 
-cat << EOM
+QUEUE="keno-web-build-agent"
```

Comment:
Why this is in common framework? It should be in the project repo

Reply:
this is the agent que that the CI pipeline uses.

Reply:
Change to use a buildkite variable, like `QUEUE=$QUEUE`, and set the variable in buildkite settings, so we do not expose `keno` in the public npm

Reply:
@codetinkerhack tagging E for his thoughts

Reply:
oh wait we already define it in buildkite so yeah not required lol

Reply:
fixed! Thanks :)